odbc_catalog.c

```diff
@@ -2307,12 +2348,21 @@ make_table_result_set (ODBC_STATEMENT * stmt, int req_handle, int type_option)
 	  goto error;
 	}
 
+	  tablename = cci_value.str;
+	  if (stmt->conn->single_schema)
+	    {
+              if (_strnicmp(tablename, stmt->conn->user, strlen(stmt->conn->user)))
```

Comment:
For avoiding wrong compare, you should split user string from tablename and compare user string and connected user.

Reply:
Because I used windows version of **strncmp** (_strnicmp), it will compare string up to the string length of connected user from table name. I think it is sufficient to identify owner of the table. Isn't it?

Reply:
If tablename is "user1.a" and connected user is "user", I think that the result of _strnicmp() will be 0. 
Then I think that it can get wrong result.

Reply:
I see. That could happen.
I will fix it.